Packs/SkyboxSecurity/Integrations/SkyboxTicketsService/SkyboxTicketsService.py

```diff
@@ -0,0 +1,2281 @@
+
+
+def resolve_datetime(input) -> Dict:
+    output = {}
+    for key in input:
+        if isinstance(input[key], datetime):
+            output[key] = input[key].isoformat()
+        else:
+            output[key] = input[key]
+    return output
+
+
+def get_cache_path():
+    path = tempfile.gettempdir() + "/zeepcache"
+    try:
+        os.makedirs(path)
+    except OSError:
+        if os.path.isdir(path):
+            pass
+        else:
+            raise
+    db_path = os.path.join(path, "cache.db")
+    try:
+        if not os.path.isfile(db_path):
+            static_init_db = os.getenv('ZEEP_STATIC_CACHE_DB', '/zeep/static/cache.db')
+            if os.path.isfile(static_init_db):
+                demisto.debug(f'copying static init db: {static_init_db} to: {db_path}')
+                shutil.copyfile(static_init_db, db_path)
+    except Exception as ex:
+        # non fatal
+        demisto.error(f'Failed copying static init db to: {db_path}. Error: {ex}')
+    return db_path
+
+
+def getTicketWorkflow_command(client, args):
+    ticketId = args.get('ticketId')
+
+    response = client.service.getTicketWorkflow(ticketId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketWorkflow',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getOriginalChangeRequestRouteInfoV1_command(client, args):
+    ticketId = args.get('ticketId')
+    changeRequestId = args.get('changeRequestId')
+
+    response = client.service.getOriginalChangeRequestRouteInfoV1(ticketId, changeRequestId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getOriginalChangeRequestRouteInfoV1',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getTicketTypePhasesByTicketType_command(client, args):
+    ticketType = args.get('ticketType')
+
+    response = client.service.getTicketTypePhasesByTicketType(ticketType)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketTypePhasesByTicketType',
+        outputs_key_field='id',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getOriginalChangeRequestV7_command(client, args):
+    ticketId = args.get('ticketId')
+
+    response = client.service.getOriginalChangeRequestV7(ticketId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getOriginalChangeRequestV7',
+        outputs_key_field='',
+        readable_output=tableToMarkdown("Original Change Request", serialize_object_list(response)),
+        outputs=serialize_object_list(response),
+        raw_response=serialize_object_list(response)
+    )
+
+    return command_results
+
+
+def setTicketFields_command(client, args):
+    ticketId = args.get('ticketId')
```

Comment:
As we are changing the support for this integration to `partner`
All arguments should be with in the following format 
```suggestion
    ticket_Id = args.get('ticket_Id')
```

Reply:
Fixed in commit `2d7f82988ad5a4a1b1394bfc9c4d0407ae880fa9`